packages/services/f-metadata/README.md

```diff
@@ -19,25 +19,29 @@ This package provides an interface for initialising Braze and handling content c
 
 ## Usage
 
+
 ### Installation
 
 Install the module using NPM or Yarn:
 
-```bash
-// yarn
-$ yarn add @justeat/f-metadata
+```sh
+yarn add @justeat/f-metadata
+```
 
-// or npm
-$ npm install @justeat/f-metadata
+```sh
+npm install @justeat/f-metadata
 ```
 
-The package can then be imported into your application:
+### Vue Applications
+
+You can import it in your Vue SFC like this (please note that styles have to be imported separately):
```

Comment:
This isn't relevant, as f-metadata has no dependency on or relation to Vue - it just happens to be only consumed by Vue components.